src/api/client.js

```diff
@@ -12,7 +12,7 @@ export function pingStudioHome() {
   );
 }
 
-export function getAssets(courseId, { page = 0, pageSize = 50, sort = 'sort', assetType = '' }) {
+export function fetchAssets(courseId, { page = 0, pageSize = 50, sort = 'sort', assetType = '' }) {
```

Comment:
Could you clarify the diffence between getAssets and fetchAssets? They sound pretty similar in their intent. 

Is fetchAssets the action creator and getAssets the reducer? Could there be a way to make it more clear the difference between the two?

Reply:
`fetchAssets` is the method in the API client that actually performs the low level `fetch` against the API; `getAssets` is the action creator itself which fires `fetchAssets` and then handles its response. I named this method `fetchAssets` so it wouldn't get confused with `getAssets` (and also so there wouldn't be a naming collision when I imported it within `src/data/actions/assets.js`).

I agree that these names aren't great, though. How's this solution?
* rename `getAssets` to `requestAssets`
* rename `assetsResponse` to `requestAssetsSuccess`
* rename `ASSETS_RESPONSE` to `REQUEST_ASSETS_SUCCESS`
* rename `fetchAssets` back to `getAssets`

As we scale up the number of XHRs we make, we will need to think about genericizing some of this machinery, but that'll need to be tackled in a followon PR.

Reply:
Whatever we end up deciding (I personally don't have a strong preference, @arizzitano's last suggestion seems good), this definitely needs to be documented and consistent across all the components we end up building.

Reply:
Let's discuss in the arch meeting today!